src/content/ai-toolkit/chat-client-sample.md

```diff
@@ -7,13 +7,11 @@ prev:
   path: /ai-toolkit/custom-llm-providers
 ---
 
-The AI Chat sample is meant to be a full-fledged chat app
-built using the Flutter AI Toolkit and Vertex AI for Firebase.
-In addition to all of the multi-shot, multi-media,
-streaming features that it gets from the AI Toolkit,
-the AI Chat sample shows how to store and manage
-multiple chats at once in your own apps.
-On desktop form-factors, the AI Chat sample looks like the following:
+The AI Chat sample is meant to be a full-fledged chat app built using the
+Flutter AI Toolkit and Vertex AI for Firebase. In addition to all of the
```

Comment:
Please, use Firebase AI Logic as name of the product. Vertex AI for Firebase is no longer a Firebase product name.